frontends/mit-open/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -0,0 +1,284 @@
+    },
+  },
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "190px",
+    borderRadius: "8px 0 8px 0",
+    backgroundColor: theme.custom.colors.darkBlue,
+  },
+})
+
+const TestimonialCardQuote = styled.div({
+  width: "648px",
+  height: "326px",
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.black,
+  padding: "0 32px 32px",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  alignItems: "flex-start",
+  flex: "1 0 0",
+  alignSelf: "stretch",
+  borderRadius: "8px",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "161px",
+    borderRadius: "0 8px 0 8px",
+    marginTop: "16px",
+    marginBottom: "16px",
+    padding: "0 16px",
+  },
+
+  ["div.testimonial-quote-opener"]: {
+    color: theme.custom.colors.mitRed,
+    fontStyle: "normal",
+    height: pxToRem(80),
+    width: "100%",
+    fontSize: pxToRem(80),
+    lineHeight: pxToRem(120),
+    [theme.breakpoints.down("md")]: {
+      fontSize: pxToRem(80),
+      height: pxToRem(50),
+      lineHeight: "normal",
+    },
+  },
+
+  ["h4"]: {
+    flexGrow: "1",
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      ...theme.typography.subtitle1,
+    },
+  },
+
+  ["div.testimonial-quote-closer"]: {
+    textAlign: "right",
+    width: "100%",
+  },
+})
+
+const TestimonialFadeRight = styled.div({
+  width: "972px",
+  height: "414px",
+  position: "absolute",
+  right: "0",
+  bottom: "0",
+  ["div"]: {
+    marginLeft: "auto",
+    height: "414px",
+    width: "246px",
+    background:
+      "linear-gradient(90deg,rgb(117 0 20 / 0%) 0%,rgb(117 0 20 / 95%) 100%)",
+  },
+  [theme.breakpoints.down("md")]: {
+    display: "none",
+  },
+})
+
+const TestimonialFadeLeft = styled.div({
+  width: "972px",
+  height: "414px",
+  position: "absolute",
+  left: "0",
+  ["div"]: {
+    height: "414px",
+    width: "246px",
+    background:
+      "linear-gradient(270deg,rgb(117 0 20 / 0%) 0%,rgb(117 0 20 / 95%) 100%)",
+  },
+  [theme.breakpoints.down("md")]: {
+    display: "none",
+  },
+})
+
+const TestimonialsSection: React.FC = () => {
+  return (
+    <Section>
+      <Container id="hamster-noises">
+        <Typography variant="h2">From our Community</Typography>
+        <Typography variant="h3">
+          Here's what other subscribers had to say about MIT Open
+        </Typography>
+        <TestimonialsDataCarouselStyled>
+          {({ resources }) => (
+            <>
+              <TestimonialsCarouselStyled
+                pageSize={1}
+                pageLeftIcon={<RiArrowDropLeftLine />}
+                pageRightIcon={<RiArrowDropRightLine />}
+                buttonAlignment="center"
+                buttonVariant="inverted"
+                buttonSize="large"
+                wrapMode="wrap"
+                scrollDistance={"slide"}
+              >
+                <TestimonialFadeLeft>
+                  <div></div>
```

Comment:
Could the gradient but on `TestimonialFadeLeft` itself rather than on the child div? I didn't understand why it was this way.